scripts/skills/perks/perk_rf_decisive.nut

```diff
@@ -0,0 +1,160 @@
+this.perk_rf_decisive <- ::inherit("scripts/skills/skill", {
+	m = {
+		// Config
+		FatigueMult = 0.85,
+		ResolveModifier = 15,
```

Comment:
I much prefer `Modifier` for anything that is added, if there is even the slightest chance it could be made negative (e.g. by some submod). It's a more neutral term and should give the guarantee that negative values will work with it. `Add` is a bit too close to `positive`

For values which must be positive or where it doesn't make sense if they are negative (weight, wage), we can use other terms like `Bonus`

Reply:
Ok I am fine with that. I will also use `Modifier` for things which are added and `Mult` for things which are multiplied.